Setup/UpgradeData.php

```diff
@@ -236,7 +236,7 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
                 );
 
                 // Set a new 'MyParcel options' group and place the option 'myparcel_fit_in_mailbox' standard on false by default
-                if (version_compare($context->getVersion(), '3.0.1', '<=')) {
+                if (version_compare($context->getVersion(), '3.0.0', '<=')) {
```

Comment:
momenteel zitten we op versie 4.0.0 dus dat zou betekenen dat deze migratie niet zal draaien met de laatste versie